src/plugin/periph/mkfatimage16.c

```diff
@@ -347,6 +347,7 @@ int main(int argc, char *argv[])
       break;
     default:
       usage();
+      free(volume_label);
```

Comment:
Shouldn't that be in a `case 'l'`
block above, where it is actually set?

Reply:
No, option 'l' creates volume_label which is used as far down as line 538

Reply:
Yes, I see...
So you probably need to
also free `bootsect_file`?

Reply:
Yes, and I just noticed there are other potential return points too. I'll drop it for now and have a rethink.